src/charm.py

```diff
@@ -110,10 +101,10 @@ def _configure_sdcore_smf(
         """Adds pebble layer and manages Juju unit status.
```

Comment:
The type hint in the method definition for event should be `EventBase`

Reply:
Changed.